packages/stryker/src/ConfigValidator.ts

```diff
@@ -34,6 +34,21 @@ export default class ConfigValidator {
     }
   }
 
+  private validateMutator() {
+    const mutator = this.strykerConfig.mutator;
+    if (typeof mutator !== 'string' && !_.isObject(mutator)) {
```

Comment:
This `if` can be rewritten as:

```ts
const mutatorType = typeof mutator;
if(mutatorType === 'object') {
 // object validation
} else if(mutatorType !== 'string'){
 // Invalidate because not supported.
}
```